app/models/transaction.rb

```diff
@@ -13,10 +13,14 @@
 
 class Transaction < ActiveRecord::Base
   CURRENCIES = %w(USD RUB)
-  TRANSACTION_TYPES = %w(Residue)
+  TRANSACTION_TYPES = %w(Residue Receipt Transfer)
+
+  attr_accessor :comission
 
   belongs_to :category, inverse_of: :transactions
   belongs_to :bank_account, inverse_of: :transactions
+  belongs_to :bank_account, inverse_of: :transactions
+  belongs_to :reference, class_name: 'BankAccount', inverse_of: :transactions
```

Comment:
@Gataniel it's not inverse of `transactions`. this association doesn't has any inverse association